zerver/lib/actions.py

```diff
@@ -294,7 +294,8 @@ def notify_invites_changed(user_profile: UserProfile) -> None:
     event = dict(type="invites_changed")
     admin_ids = [user.id for user in
                  user_profile.realm.get_admin_users_and_bots()]
-    send_event(user_profile.realm, event, admin_ids)
+    send_event(user_profile.realm, event, admin_ids,
+               description="For realm administrators when invitations are sent")
```

Comment:
Maybe this should be extracted to a different commit?

Reply:
It's a single use of a new feature; I think it probably belongs just to show how this is supposed to work.